packages/dcos-integration-test/extra/test_metrics.py

```diff
@@ -35,6 +35,18 @@ def expected_datapoint_response(response):
         """
         assert 'datapoints' in response, '"datapoints" dictionary not found'
         'in response, got {}'.format(response)
+
+        for dp in response['datapoints']:
+            assert 'name' in dp, '"name" parameter should not be empty, got {}'.format(dp)
+            if 'filesystem' in dp['name']:
+                assert 'tags' in dp, '"tags" key not found, got {}'.format(dp)
+
+                assert 'path' in dp['tags'], '"path" tag not found for filesystem metric, '
+                'got {}'.format(dp)
```

Comment:
i am quite surprised pep8 did not catch the indentation here
https://www.python.org/dev/peps/pep-0008/#indentation